packages/api-client-core/spec/operationRunners.spec.ts

```diff
@@ -751,6 +853,95 @@ describe("operationRunners", () => {
       expect(results[1].name).toBeTruthy();
     });
 
+    test("can run a bulk action with a returnType", async () => {
+      const promise = actionRunner<{ id: string; name: string }>(
+        {
+          connection,
+        },
+        "bulkCreateWidgets",
+        { id: true, name: true },
+        "widget",
+        "widgets",
+        true,
+        {
+          inputs: {
+            value: [{ id: "123", widget: { name: "foo" } }],
+            required: true,
+            type: "[BulkCreateWidgetInput!]",
+          },
+        },
+        {},
+        null,
+        true
+      );
+
+      mockUrqlClient.executeMutation.pushResponse("bulkCreateWidgets", {
+        data: {
+          bulkCreateWidgets: {
+            success: true,
+            errors: null,
+            results: [{ hello: "world" }, { hello: "other world" }],
+          },
+        },
+        stale: false,
+        hasNext: false,
+      });
+
+      const results = await promise;
+
+      expect(results[0]).toEqual({ hello: "world" });
+      expect(results[1]).toEqual({ hello: "other world" });
```

Comment:
```suggestion
            results: [{ hello: "world" }, "some scalar"],
          },
        },
        stale: false,
        hasNext: false,
      });

      const results = await promise;

      expect(results[0]).toEqual({ hello: "world" });
      expect(results[1]).toEqual("some scalar");
```

just to exercise the code path to make sure folks that return non-objects as their return type will be ok

Reply:
👍